languages/javascript/reference/keywords/export.md

```diff
@@ -0,0 +1,29 @@
+# `export`
+
+The `export` keyword is used to define elements of the current file, that can be imported into other files using the [`import`][concept-import] keyword.
```

Comment:
```suggestion
The `export` keyword is used to define elements of the current file, that can be imported into other files using the [`import`][keyword-import] keyword.
```

I know this is pedantic, but it will help if you have both a concept and a keyword with the same name :)

Reply:
I like pedantic :D